### ISSUE TITLE
Enhancement - voice control for climates ?

### ISSUE THREAD

**Issue by pbn42** [t0bst4r/home-assistant-matter-hub#222]
Hi,

I'm using your extention with Alexa, and everything works fine, except for climate voice controls (all versions to date).

(i'm french, so i'll try to translate the messages)

My climate is integrated on Alexa using :
- the original Gree skill (climate bureau)
- your extention (climate test)

Using the voice, and the gree name, i can tell "Alexa, turn on climate bureau", or "Alexa, turn off climate bureau", or "Alexa, set climate bureau to 22°", and everything works fine.

When i'm trying using your integration, Alexa answers me "The device climate test doesn't support this command"

Here is my climate :

![image](https://github.com/user-attachments/assets/65ee7abf-3c6a-43d2-bea8-fc561bd3e0ca)

Thanks a lot !

**Comment by alexh8555** [comment]
This happens to me as well.
I'm using matterhub with Google Home, I can see the climate device on nest hub and mobile app.
Climate devices are controlable by both nest hub's touch screen and mobile app.
But when I tried to use voice control to turn it on, it says device is not setup yet.
Devices other than climate are working correctly.

**Comment by Apollon77** [comment]
Did you checked if Alexa supports Matter Climare devcies already? You can not assume that atter device types are supported just because the "Alexa Skills" support such a device type.

**Comment by t0bst4r** [comment]
Yes, climates are supported - at least we can see them and control it via the app.
At the moment, climates / thermostats are somehow buggy, since we need to find out, how it actually works and what behaves differently between HA and matter.
